slither/printers/summary/human_summary.py

```diff
@@ -291,6 +267,36 @@ def _get_features(self, contract):  # pylint: disable=too-many-branches
             "Proxy": contract.is_upgradeable_proxy,
         }
 
+    def _get_contracts(self, txt):
```

Comment:
This and the next function were pulled out of `output` because I was getting a "too many branches" error from the linter.